src/components/TweetContainer/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import { FiHeart } from 'react-icons/fi';
+import { avatar } from '@/constants/images';
+import { TweetPropsType } from '@/components/TweetContainer/type';
+import { useCount } from '@/hooks/useCount';
+
+export const TweetContainer: FC<TweetPropsType> = ({
+  tweet,
+  name,
+  nickName,
+  date,
+  count,
+}) => {
+  const { onToggle, counter, isPressed } = useCount(count, false);
```

Comment:
I don't really think there's a use for such hook, it has very simple logic, also I want you to update the amount of likes in the application state, not just visually.